app/containers/ActionSheet/ActionSheet.tsx

```diff
@@ -140,6 +140,8 @@ const ActionSheet = React.memo(
 						style={{ ...styles.container, ...bottomSheet }}
 						backgroundStyle={{ backgroundColor: colors.focusedBackground }}
 						onChange={index => index === -1 && onClose()}
+						activeOffsetY={[-5, 5]}
```

Comment:
Why was it necessary to add this? Have you validated if it impacts in other places?

Reply:
Without adding this, bottom sheet gestures didn't work (Couldn't change snap). I did test the other sheets on android.